rosdep/python.yaml

```diff
@@ -373,13 +373,18 @@ python-clearsilver:
     saucy: [python-clearsilver]
     trusty: [python-clearsilver]
 python-click:
+  debian:
+    sid: [python-click]
+    stretch: [python-click]
   fedora: [python-click]
   osx:
     pip:
       packages: [click]
   ubuntu:
     pip:
       packages: [click]
+    xenial: [python-click]
```

Comment:
I don't believe rosdep supports mixed level rules. With this change the pip rule should be defined for all the older platforms.